OpenNMS-Cloud/lokahi
# HS-1564: Automation: UI: Flows

This PR contains:

1. support enabling and disabling welcome wizard per test. By default any tests with @cloud should skip the wizard
2. left nav bar issue when its not hiding
3. tests are using env variables: 
            CLOUD_MINION_GATEWAY_HOST
            CLOUD_INSTANCE_URL
            CLOUD_USERNAME
            CLOUD_PASSWORD
            IS_CLOUD_TESTING_TARGET
4. basic flows test:
        - create location
        - download cert from location page
        - start minion with SNMP and UDPGen
        - create new discovery from discovery page
        - check node discovered and monitored
        - check flow data visible on home page
        - check flow data visible in the flow page
